contrib/gorm.io/gorm.v1/gorm.go

```diff
@@ -33,6 +33,21 @@ const (
 	gormSpanStartTimeKey = key("dd-trace-go:span")
 )
 
+type GormTracePlugin struct{}
```

Comment:
Can you add options as struct field so they can be passed to `withCallbacks`?

Reply:
Done